native/cocos/bindings/event/EventDispatcher.h

```diff
@@ -115,6 +116,66 @@ class TouchEvent : public OSEvent {
     Type type = Type::UNKNOWN;
 };
 
+enum class StickKeyCode {
+    UNDEFINE = 0,
+    A,
+    B,
+    X,
+    Y,
+    L1,
+    R1,
+    MINUS,
+    PLUS,
+    L3,
+    R3,
+    MENU,
+    START,
+    LEFT_STICK,
+    RIGHT_STICK,
+};
+
+enum class StickAxisCode {
+    UNDEFINE = 0,
+    X,
+    Y,
+    LEFT_STICK_X,
+    LEFT_STICK_Y,
+    RIGHT_STICK_X,
+    RIGHT_STICK_Y,
+    L2,
+    R2,
+    LEFT_GRIP,
+    RIGHT_GRIP,
+};
+
+struct ControllerInfo {
+    struct AxisInfo {
+        StickAxisCode axis = StickAxisCode::UNDEFINE;
+        float         value = 0.0F;
+        AxisInfo(StickAxisCode axis, float value) : axis(axis), value(value) {}
+    };
+    struct ButtonInfo {
+        StickKeyCode key;
+        bool         isPress;
+        ButtonInfo(StickKeyCode key, bool isPress) : key(key), isPress(isPress) {}
+    };
+
+    int napdId;
```

Comment:
default value

Reply:
This is the gamepad adaptation code synchronized from the NX branch. It has not yet been synchronized to 3.6.0, but we need to use it. Do I need to modify it?